app/javascript/scss/elements/_dropdown.scss

```
@@ -51,6 +51,11 @@
     right: auto;
   }
 
+  &--center {
+    right: 50%;
+    translate: 50% 0;
```

`translate` isn't well enough supported (we supposed Safari 13+)

```suggestion
    transform: translateX(50%);
```